src/cbc_sdk/endpoint_standard/base.py

```diff
@@ -330,6 +351,40 @@ def _get_detailed_results(self):
                 self._info = results[0]
                 return self
 
+    def ban_process_sha256(self, description=""):
+        """Bans the application by adding the process_sha256 to the BLACK_LIST
```

Comment:
Flag this for changing once the API moves to different terminology. Likewise with WHITE_LIST. Otherwise no action needed at this time.